### ISSUE TITLE
Exception on closing a RubyMine project with an open quokka scratch file

### ISSUE THREAD

**Issue by csterritt** [wallabyjs/quokka#46]
### Issue description or question
When I closed a RubyMine project that had an open quokka scratch file, I got the following exception backtrace:

```
java.lang.NullPointerException
	at com.intellij.ui.content.impl.ContentManagerImpl.c(ContentManagerImpl.java:171)
	at com.intellij.ui.content.impl.ContentManagerImpl.addContent(ContentManagerImpl.java:150)
	at com.quokka.javascript.QuokkaView.a(Unknown Source)
	at com.quokka.javascript.session.j.run(Unknown Source)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:303)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:410)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:399)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.f(IdeEventQueue.java:827)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:655)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:365)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

### Code editor version

RubyMine 2017.1.2

### OS name and version

OSX 10.12.4

**Comment by ArtemGovorov** [comment]
Thanks for the report. The issue is fixed and the fix is published in the Quokka plugin v1.0.5.

**Comment by csterritt** [comment]
Excellent, thanks!
